### PR TITLE
split/bud orbital myositis from orbital pseudotumor

### LINKED ISSUE #7257: MONDO:0004769 orbital plasma cell granuloma- Split this Mondo ID?
**Mondo term (ID and Label):**
MONDO:0004769 orbital plasma cell granuloma

**Xref that should be fixed (ID and label):**
Some x-refs/exact matches on this term map to a concept with a different name/CUI in UMLS, Not sure if these are different, or if UMLS has split unnecessarily?
http://identifiers.org/snomedct/80698001 is "orbital myositis"
[Orbital Myositis](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FNCIT_C117296?lang=en)[NCIT]  C117296  "Rare form of myositis that affects only the orbital muscles."
(See also  MESH D055622) All these map to C2350476

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**
I am wondering about the pref name on this term, given the other exact matches:
[mesh/D016727](http://identifiers.org/mesh/D016727) is "Orbital pseudotumor" with syn of "Inflammatory Pseudotumor of Orbit"
SNOMEDCT [72789009](http://identifiers.org/snomedct/72789009) is "Inflammatory pseudotumor of orbit"  (both of which are on C0085270 in UMLS)
(Note also that UMLS has split Plasma Cell Granuloma, Orbital into a separate CUI: C2350387, so even though MeSH uses all of these names on D016727, that is split across multiple CUIs)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7257

@twhetzel please see discussion on the ticket.

Could you also verify that my logical definition is correct on orbital myositis (according to google, the extraocular muscles are the muscles of the orbit)

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -514209,6 +514208,21 @@ is_a: MONDO:0859689 {source="https://orcid.org/0000-0001-8134-3037"} ! hepatobil
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7300" xsd:anyURI
 
+[Term]
+id: MONDO:0100550
+name: orbital myositis
+def: "A rare form of myositis that affects only the orbital muscles." [NCIT:C117296]
+synonym: "ocular myositis" EXACT [https://orcid.org/0000-0002-3458-4839]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:514215]: @nicolevasilevsky based on researching to confirm the logical definition, I'd like to propose adding this synonym.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:514215]: sounds good to me, thanks!

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -514209,6 +514208,21 @@ is_a: MONDO:0859689 {source="https://orcid.org/0000-0001-8134-3037"} ! hepatobil
...
+xref: ICD9:376.12 {source="MONDO:equivalentTo", source="MONDO:i2s"}
+xref: NCIT:C117296 {source="MONDO:equivalentTo"}
+xref: SCTID:80698001 {source="MONDO:equivalentTo"}
+xref: UMLS:C2350476 {source="NCIT:C117296", source="MONDO:notFoundInDiseaseSubset"}
+is_a: MONDO:0021167 {source="NCIT:C117296", source="https://orcid.org/0000-0001-5208-3432"} ! myositis disease
+intersection_of: MONDO:0021167 ! myositis disease
+intersection_of: disease_has_location UBERON:0001601 ! extra-ocular muscle
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:514222]: I agree with the logical definition.

**Review by twhetzel — APPROVED** [review]
Looks good! I add a synonym - can you confirm? 

I can wait for QC checks to finish and merge after you confirm if you prefer - just re-assign it back to me.

**Comment by nicolevasilevsky** [comment]
your synonym looks good, all the checks have passed, I will merge. Thank you!

**Pull request merged**